app/views/taxons/_page_header.html.erb

```diff
@@ -2,7 +2,7 @@
   <div class="full-page-width-wrapper">
     <%= render partial: 'govuk_component/breadcrumbs',
       locals: {
```

Comment:
Shouldn't this also render `govuk_publishing_components/components/contextual_breadcrumbs`, in the same way as the `breadcrumbs` partial?